<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgExtensionConf</class>
 <widget class="QDialog" name="DlgExtensionConf">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>463</width>
    <height>337</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Extension Configuration</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>381</width>
     <height>308</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QFormLayout" name="formExtensionConf">
      <item row="0" column="0">
       <widget class="QLabel" name="lvlExtension">
        <property name="font">
         <font>
          <family>Cantarell</family>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Extension</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="leExtionsion"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lblName">
        <property name="font">
         <font>
          <family>Cantarell</family>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Name</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="leName"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lblPassword">
        <property name="font">
         <font>
          <family>Cantarell</family>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Password</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lePassword"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="lblContext">
        <property name="font">
         <font>
          <family>Cantarell</family>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Context</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="leContext"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="lblCodec">
        <property name="font">
         <font>
          <family>Cantarell</family>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Codecs</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="lblMedEnc">
        <property name="font">
         <font>
          <family>Cantarell</family>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Media Encryption</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="lblAdminMode">
        <property name="font">
         <font>
          <family>Cantarell</family>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Admin Mode</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="lblRecMode">
        <property name="font">
         <font>
          <family>Cantarell</family>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Recording Mode</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="leCodec"/>
      </item>
      <item row="5" column="1">
       <widget class="QComboBox" name="cbMediaEncry"/>
      </item>
      <item row="6" column="1">
       <widget class="QComboBox" name="cbAdminMode"/>
      </item>
      <item row="7" column="1">
       <widget class="QComboBox" name="cbRecMode"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="pbClear">
        <property name="font">
         <font>
          <family>Cantarell</family>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Clear</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbDefault">
        <property name="font">
         <font>
          <family>Cantarell</family>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Default</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbSave">
        <property name="font">
         <font>
          <family>Cantarell</family>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbCancel">
        <property name="font">
         <font>
          <family>Cantarell</family>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pbCancel</sender>
   <signal>clicked()</signal>
   <receiver>DlgExtensionConf</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>370</x>
     <y>308</y>
    </hint>
    <hint type="destinationlabel">
     <x>429</x>
     <y>315</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
